tests/codemods/test_flask_json_response_type.py

```diff
@@ -0,0 +1,238 @@
+from core_codemods.flask_json_response_type import FlaskJsonResponseType
+from tests.codemods.base_codemod_test import BaseCodemodTest
+from textwrap import dedent
+
+
+class TestFlaskJsonResponseType(BaseCodemodTest):
+    codemod = FlaskJsonResponseType
+
+    def test_name(self):
+        assert self.codemod.name() == "flask-json-response-type"
+
+    def test_simple(self, tmpdir):
+        input_code = """\
+        from flask import make_response, Flask
+        import json
+
+        app = Flask(__name__)
+
+        @app.route("/test")
+        def foo(request):
+            json_response = json.dumps({ "user_input": request.GET.get("input") })
+            return make_response(json_response)
+        """
+        expected = """\
+        from flask import make_response, Flask
+        import json
+
+        app = Flask(__name__)
+
+        @app.route("/test")
+        def foo(request):
+            json_response = json.dumps({ "user_input": request.GET.get("input") })
+            return (make_response(json_response), {'Content-Type': 'application/json'})
```

Comment:
Wouldn't the simplest solution in this case to be to just return `json_response` since Flask will automatically set the proper context type?

Reply:
Actually, according to the `flask` documentation, `make_response` will also correctly handle the content type by calling `jsonify` on a `dict`: https://flask.palletsprojects.com/en/3.0.x/api/#flask.Flask.make_response